core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowKeys.scala

```diff
@@ -75,7 +75,10 @@ trait CloudflowTaskKeys {
     taskKey[Map[String, Config]]("Streamlets found in sub projects by scanning the application classpath.")
 
   private[sbt] val cloudflowStreamletDescriptors = taskKey[Map[String, Config]]("Streamlets found by scanning the application classpath.")
-  private[sbt] val cloudflowApplicationClasspath = taskKey[Array[URL]]("classpath of the user project.")
+  private[sbt] val cloudflowStreamletDescriptorsByProject =
```

Comment:
This looks the same as `allStreamletDescriptorsByProject`?

Reply:
one is needed to populate the other.